portal-impl/src/portal.properties

```diff
@@ -131,7 +131,8 @@
         com.liferay.portal.upgrade.UpgradeProcess_6_1_1\,\
         com.liferay.portal.upgrade.UpgradeProcess_6_2_0\,\
         com.liferay.portal.upgrade.UpgradeProcess_7_0_0\,\
-        com.liferay.portal.upgrade.UpgradeProcess_7_0_1
+        com.liferay.portal.upgrade.UpgradeProcess_7_0_1\,\
+        com.liferay.portal.upgrade.UpgradeProcess_7_0_2
```

Comment:
Hey @brianchandotcom, @sergiogonzalez, 

Have you made a decision about the next version for the core in master?
https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/util/ReleaseInfo.java#L151
I guess it should be 7100 or 8000 as we did when we released 6.2

Thanks.